src/com/nutrons/steamworks/Drivetrain.java

```diff
@@ -255,8 +255,8 @@ public Command driveTimeAction(long time) {
    */
   public Command driveTeleop() {
     return driveHoldHeading(
-        combineLatest(throttle, yaw, (x, y) -> x + y).publish().autoConnect().onBackpressureDrop(),
-        combineLatest(throttle, yaw, (x, y) -> x - y).publish().autoConnect().onBackpressureDrop(),
+        combineLatest(throttle, yaw, (x, y) -> x + y).map(x -> Math.abs(x) * x).publish().autoConnect().onBackpressureDrop(),
+        combineLatest(throttle, yaw, (x, y) -> x - y).map(x -> Math.abs(x) * x).publish().autoConnect().onBackpressureDrop(),
```

Comment:
Make the maps to throttle and yaw happen in the constructor.
i.e. `this.throttle = throttle.map(x -> Math.abs(x) * x) `
that way it will apply everywhere we use the throttle and yaw.
Also, you may want to make the map something like `this.throttle = throttle.map(x -> 1.1 * Math.abs(x) * x) `
in order to make sure when we push joystick to the max, which may be like 0.95, it doesn't become smaller, and instead pushes to our maximum (1.0). But this is minor and idk if it actually makes a difference / matters / would work.